### PR TITLE
photo feed

### PR DESCRIPTION
**Pull request by emirodriguezalc**

### CODE REVIEW

File: js-client/public/photosFeed/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Photos Home</title>
```
Comment (carlos-ch) [inline js-client/public/photosFeed/index.html:7]: title: photos feed

**Review by carlos-ch — CHANGES_REQUESTED** [review]
just one comment

File: js-client/public/photosFeed/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-
+const UNSPLASH_TOKEN = GpKqhzxWheY1xmlfCvT_a8dXzNeQgBeA1i6wptdXBbk
```
Comment (carlos-ch) [inline js-client/public/photosFeed/index.js:1]: wrap in strings

**Review by carlos-ch — CHANGES_REQUESTED** [review]
one comment

File: js-client/public/photosFeed/index.js (hunk 2 above)
Comment (emirodriguezalc) [reply js-client/public/photosFeed/index.js:1]: Damn

**Review by carlos-ch — APPROVED** [review]
all good!

**Pull request merged**